public/javascripts/Admin/src/Admin.GSVCommentView.js

```diff
@@ -0,0 +1,68 @@
+function AdminGSVCommentView(admin) {
+    var self = {};
+    self.admin = admin;
+
+    var _init = function() {
+        self.panoProp = new PanoProperties();
+
```

Comment:
I don't think the extra line needs to be here. The function is only 2 lines long, separating out the different sections with the extra line seems like overkill :wink: